kotlinx.fuzz.jazzer/build.gradle.kts

```diff
@@ -4,6 +4,10 @@ plugins {
     id("kotlinx.fuzz.src-module")
 }
 
+repositories {
+    mavenLocal()
+}
```

Comment:
Unnecessary, our plugin `kotlinx.fuzz.src-module` alredy added it.

Reply:
Fixed